integration/test/app/basic/simple_test.go

```diff
@@ -50,6 +49,11 @@ func TestAppLifecycle(t *testing.T) {
 			if err != nil {
 				t.Fatalf("expected %#v got %#v", nil, err)
 			}
+
+			// TODO: Removing hardcoding once there is a chart-operator release
+			// with Helm 3 support in the default catalog.
+			tarballURL = "https://giantswarm.github.io/default-test-catalog/chart-operator-0.12.1-13521d4e2cb5378dbff26995e094d1c23a15e121.tgz"
+
```

Comment:
Like the TODO says this hardcoding is needed until there is a release of chart-operator with helm 3 support in the catalog.